machineconfiguration/v1/tests/controllerconfigs.machineconfiguration.openshift.io/VSphereMultiVCenters.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Shamefully missing"
```

Comment:
Can we make this more appropriate to the feature/types, see how we format this for other tests

Reply:
This test file confused me.  For some reason the tooling said I had to add it when I didn't make a direct change to it.  I copied the same test as VSphereControlPlaneMachineSet.yaml. In fact, reviewing all the other tests for controllerconfigs, they are the same copy.

Reply:
I updated the test name the new convention.  We'll need to update the rest of the tests in that directory at a later time.